dev/pragma.h

```diff
@@ -152,6 +186,24 @@ namespace sqlite_orm {
           private:
             friend struct storage_base;
 
+            bool foreign_keys() {
+                return this->get_pragma<bool>("foreign_keys");
+            }
+            void foreign_keys(bool value) {
+                /*
+                 * this define makes it impossible to change the value of FK checking if undefined!!
+                 */
+#ifdef FK_TOGGLE_ENABLE
```

Comment:
this is not acceptable cause if `FK_TOGGLE_ENABLE` is going to be used only by you then leave it in your local branch. No need to commit things which are never used. We don't pay for what we don't use.

Reply:
I would like to keep this minimum support to FK toggling as part of the repo because we can document it as the configuration that tries to preserve data by making a backup when possible
It is not only for me, it is a configurable option

Reply:
you might learn to be less selfish and reach a compromise with your collaborators who work very hard to make a better product...a very few lines makes the product configurable and would satisfy the people who want to save data when possible.

Reply:
collaborating is making compromises, accepting others' ideas and letting those ideas enrich the product.

Reply:
I do reach compromise often enough. If I don't accept your feature it means that I don't accept your feature not I am selfish/compromise hater etc. Please look at the amount of PRs created by other people for the whole history of the lib. Most of opened PRs from other people were merged successfully. Please consider this before calling someone selfish